xmake/rules/c++/modules/get_module_flags.lua

```diff
@@ -0,0 +1,52 @@
+function get_module_flags(compinst, toolname, opt)
```

Comment:
we can define main entry `function main(compinst, toolname, opt)` if there is only one interface.

then

```lua
import("get_module_flags")

get_module_flags(compinst, toolname, opt)
```